tool/planet/agent.go

```diff
@@ -329,6 +348,26 @@ func runAgent(config agentConfig) error {
 		config.agent.Tags = make(map[string]string)
 	}
 	config.agent.Tags["role"] = string(config.monitoring.Role)
+
+	clientset, err := getKubeClientFromPath(constants.KubectlConfigPath)
```

Comment:
Be careful with kubectl's kubeconfig - it is restricted on regular nodes so you won't be able to query much.